docs/csharp/fundamentals/types/interfaces.md

```diff
@@ -1,14 +1,14 @@
 ---
 title: "Interfaces - define behavior for multiple types"
 description: An interface in C# contains definitions for a group of related functionalities that a non-abstract class or a struct must implement. It specifies the members and their signatures for a type that implements the interface.
-ms.date: 05/14/2021
+ms.date: 10/13/2025
 helpviewer_keywords: 
   - "interfaces [C#]"
   - "C# language, interfaces"
 ---
 # Interfaces - define behavior for multiple types
 
-An interface contains definitions for a group of related functionalities that a non-abstract [`class`](../../language-reference/keywords/class.md) or a [`struct`](../../language-reference/builtin-types/struct.md) must implement. An interface may define `static` methods, which must have an implementation. An interface may define a default implementation for members. An interface may not declare instance data such as fields, automatically implemented properties, or property-like events.
+An interface contains definitions for a group of related functionalities that a non-abstract [`class`](../../language-reference/keywords/class.md) or a [`struct`](../../language-reference/builtin-types/struct.md) must implement. An interface might define `static` methods, which must have an implementation. An interface might define a default implementation for members. An interface might not declare instance data such as fields, automatically implemented properties, or property-like events.
```

Comment:
This changed in a new version. Interfaces can now declare `abstract static` and `virtual static` members. Also, a couple might-> can (or can't) changes

```suggestion
An interface contains definitions for a group of related functionalities that a non-abstract [`class`](../../language-reference/keywords/class.md) or a [`struct`](../../language-reference/builtin-types/struct.md) must implement. An interface can define `static` methods. An interface can define a default implementation for members. An interface can't declare instance data such as fields, automatically implemented properties, or property-like events.
```